standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -4675,4 +4676,145 @@ public void dropStoredProcedure(StoredProcedureRequest request) throws MetaExcep
   public List<String> getAllStoredProcedures(ListStoredProcedureRequest request) throws MetaException, TException {
     return client.get_all_stored_procedures(request);
   }
+
+  @Override
+  public void addPackage(Package request) throws NoSuchObjectException, MetaException, TException {
+    client.add_package(request);
+  }
+
+  @Override
+  public Package findPackage(PackageRequest request) throws TException {
+    return client.find_package(request);
+  }
+
+  @Override
+  public List<String> listPackages(ListPackageRequest request) throws TException {
+    return client.get_all_packages(request);
+  }
+
+  @Override
+  public void dropPackage(PackageRequest request) throws TException {
+    client.drop_package(request);
+  }
+
+  /**
+   * Builder for the GetProjectionsSpec. This is a projection specification for partitions returned from the HMS.
+   */
+  public static class GetPartitionProjectionsSpecBuilder {
```

Comment:
I did not understand what these extra builder classes are for. Looks like unused.

Reply:
Looks like some rebase + resolve conflict went wrong. This came from someone else but it supposed to be deleted instead of added. I removed it from the new patch.